packages/notebook/src/searchprovider.ts

```diff
@@ -437,49 +471,53 @@ export class NotebookSearchProvider extends SearchProvider<NotebookPanel> {
     this._onSearchProviderChanged();
   }
 
-  private _onPlaceholderRendered(
-    panel: Notebook,
-    renderedCell: Cell<ICellModel>
-  ): void {
-    const index = panel.widgets.findIndex(cell => cell.id === renderedCell.id);
-    if (index >= 0) {
-      void this._onCellsChanged(panel.model!.cells, {
-        newIndex: index,
-        newValues: [renderedCell.model],
-        oldIndex: index,
-        oldValues: [renderedCell.model],
-        type: 'set'
-      });
-    }
-  }
+  // private _onPlaceholderRendered(
+  //   panel: Notebook,
+  //   renderedCell: Cell<ICellModel>
+  // ): void {
+  //   const index = panel.widgets.findIndex(cell => cell.id === renderedCell.id);
+  //   if (index >= 0) {
+  //     void this._onCellsChanged(panel.model!.cells, {
+  //       newIndex: index,
+  //       newValues: [renderedCell.model],
+  //       oldIndex: index,
+  //       oldValues: [renderedCell.model],
+  //       type: 'set'
+  //     });
+  //   }
+  // }
```

Comment:
Commented code?

Reply:
Thanks @krassowski this is dead code - I'll open a PR to remove it.


Its purpose was to add a placeholder cell to the list of cells for search. But it is not required any longer as search is happening on the model and defer rendering has changed.